templates/cisco_ios_show_mac-address-table.textfsm

```diff
@@ -18,6 +18,7 @@ TYPE1
   ^Displaying\s+entries
   ^Destination\s+Address\s+Address\s+Type\s+VLAN\s+Destination\s+Port
   ^\s*$$
+  ^Multicast Entries -> End
```

Comment:
Do you mind adding `\s+` between **Multicast Entries** rather than the static spacing. Once that is updated we can get this merged in.

Reply:
Done!